go/gas-oracle/flags/flags.go

```diff
@@ -8,13 +8,24 @@ var (
 	EthereumHttpUrlFlag = cli.StringFlag{
 		Name:   "ethereum-http-url",
 		Value:  "http://127.0.0.1:8545",
-		Usage:  "Sequencer HTTP Endpoint",
+		Usage:  "L1 HTTP Endpoint",
 		EnvVar: "GAS_PRICE_ORACLE_ETHEREUM_HTTP_URL",
 	}
-	ChainIDFlag = cli.Uint64Flag{
-		Name:   "chain-id",
+	LayerTwoHttpUrlFlag = cli.StringFlag{
+		Name:   "layer-two-http-url",
+		Value:  "http://127.0.0.1:9545",
+		Usage:  "Sequencer HTTP Endpoint",
```

Comment:
Does this have to be the Sequencer? Can it be any L2 node?

Reply:
Technically it has to be the sequencer since the sequencer is the only one that accepts transactions, in practice it can be any of the infra providers since the proxy the sequencer for `eth_sendRawTransaction`